### PR TITLE
feat: redisign choose language

### PR DESCRIPTION
**Pull request by margarita-sh**

### CODE REVIEW

File: frontend/src/app/libs/container/components/localization/localization.component.html [hunk 1]
```diff
@@ -4,9 +4,9 @@
 	</app-button>
 	<mat-menu class="localization__menu" #menu="matMenu">
 		<button mat-menu-item class="localization__menu-item"
-				*ngFor="let locale of translateService.getLangs()"
-				(click)="translateService.use(locale)">
-			<span>{{locale | uppercase}}</span>
+			*ngFor="let locale of translateService.getLangs()" (click)="translateService.use(locale)">
+		<img [src]=" locale == 'en' ? 'assets/images/localization/icon-uk-flag.svg' : 'assets/images/localization/icon-russian-flag.svg'" class="localization__flag">
+			<span class="localization__language">{{(locale == 'en' ? 'BUTTON.LANGUAGE-EN' : 'BUTTON.LANGUAGE-RU') | translate}}</span>
```
Comment (clain-atomi) [inline frontend/src/app/libs/container/components/localization/localization.component.html:9]: move `locale == 'en' ? 'BUTTON.LANGUAGE-EN' : 'BUTTON.LANGUAGE-RU') ` to getter

Comment (clain-atomi) [reply frontend/src/app/libs/container/components/localization/localization.component.html:9]: use ===

File: frontend/src/app/libs/container/components/localization/localization.component.scss [hunk 2]
```diff
@@ -5,14 +5,20 @@
...
+	@include menu-item-hover;
+	color: $color-white;
+	width:152px;
+	margin:8px 0;
   }
-
   &__menu-trigger {
     display: flex;
-    align-items: center;
+	align-items: center;
```
Comment (clain-atomi) [inline frontend/src/app/libs/container/components/localization/localization.component.scss:15]: seems like indendation here is wrong

File: frontend/src/app/libs/container/components/localization/localization.component.scss [hunk 3]
```diff
@@ -5,14 +5,20 @@
...
+  }
+  &__language {
```
Comment (clain-atomi) [inline frontend/src/app/libs/container/components/localization/localization.component.scss:17]: add new empty line above

File: frontend/src/app/libs/container/components/localization/localization.component.scss [hunk 4]
```diff
@@ -5,14 +5,20 @@
...
+	  @include text-p();
   }
+   &__flag {
```
Comment (clain-atomi) [inline frontend/src/app/libs/container/components/localization/localization.component.scss:20]: same here, please check indendation (tabs, 4 spaces), and add new empty line between selectors

File: frontend/src/app/libs/container/components/localization/localization.component.scss [hunk 5]
```diff
@@ -5,14 +5,20 @@
...
+	   margin-right: 16px;
```
Comment (clain-atomi) [inline frontend/src/app/libs/container/components/localization/localization.component.scss:21]: please use $grid-main-unit * 2;
all sizes must be represented by $grid-main/secondary-unit * x;

**Review by clain-atomi — CHANGES_REQUESTED** [review]

File: frontend/src/app/libs/container/components/localization/localization.component.html [hunk 6]
```diff
@@ -4,9 +4,9 @@
 	</app-button>
 	<mat-menu class="localization__menu" #menu="matMenu">
 		<button mat-menu-item class="localization__menu-item"
-				*ngFor="let locale of translateService.getLangs()"
-				(click)="translateService.use(locale)">
-			<span>{{locale | uppercase}}</span>
+			*ngFor="let locale of translateService.getLangs()" (click)="translateService.use(locale)">
+		<img [src]=" locale == 'en' ? 'assets/images/localization/icon-uk-flag.svg' : 'assets/images/localization/icon-russian-flag.svg'" class="localization__flag">
```
Comment (maximwow70) [inline frontend/src/app/libs/container/components/localization/localization.component.html:8]: It's a good practice to place class property at the first property of element.

File: frontend/src/app/libs/container/components/localization/localization.component.html (hunk 1 above)
Comment (maximwow70) [reply frontend/src/app/libs/container/components/localization/localization.component.html:9]: Agree with you, but also can suggest to use map pattern to get locale metadata.
In result just need to use smth like getLocaleMetadata(locale).buttonLanguage or getLocaleMetadata(locale).iconUrl.
And map can be organized in the following way: locale as key, metadata object as value.
What do you think about that?

File: frontend/src/app/libs/container/components/localization/localization.component.scss [hunk 7]
```diff
@@ -5,14 +5,20 @@
   @include drop-down-menu-container;
 
   &__menu-item {
-		@include menu-item-hover;
-		
-    text-align: center;
-    color: $color-white;
+	@include menu-item-hover;
+	color: $color-white;
```
Comment (maximwow70) [inline frontend/src/app/libs/container/components/localization/localization.component.scss:9]: The same with indentation.

**Review by clain-atomi — APPROVED** [review]

**Pull request merged**